tests/ci.yml

```diff
@@ -0,0 +1,38 @@
+.snippets:
+  mambaforge_setup:
```

Comment:
This could potentially be wrapped up in a docker image that is built one time per pipeline rather than in every job with the added bonus of a publicly available conifer, up to date, docker image that people can use. There would be some potential issue using the ci4fpga resources however